tools/profiling/memory/memory_diff.py

```diff
@@ -49,29 +49,39 @@
         (rb'server call memory usage: ([0-9\.]+) bytes per call', float),
 }
 
+_SCENARIOS = {
+    'default': [],
+    'minstack': ['--minstack'],
+}
+
 
 def _run():
     """Build with Bazel, then run, and extract interesting lines from the output."""
     subprocess.check_call([
         'tools/bazel', 'build', '-c', 'opt',
         'test/core/memory_usage/memory_usage_test'
     ])
-    output = subprocess.check_output([
-        'bazel-bin/test/core/memory_usage/memory_usage_test',
-        '--warmup=10000',
-        '--benchmark=50000',
-    ])
     ret = {}
-    for line in output.splitlines():
-        for key, (pattern, conversion) in _INTERESTING.items():
-            m = re.match(pattern, line)
-            if m:
-                ret[key] = conversion(m.group(1))
+    for scenario, extra_args in _SCENARIOS.items():
+        try:
+            output = subprocess.check_output([
+                'bazel-bin/test/core/memory_usage/memory_usage_test',
+                '--warmup=10000',
+                '--benchmark=50000',
+            ] + extra_args)
+        except subprocess.CalledProcessError as e:
+            print('Error running benchmark:', e)
+            continue
+        for line in output.splitlines():
+            for key, (pattern, conversion) in _INTERESTING.items():
+                m = re.match(pattern, line)
```

Comment:
I'm guessing this is fine, but just in case you haven't run it - is it an issue that output is encoded bytes? The `universal_newlines` argument may be needed here.